infracost-usage-example.yml

```diff
@@ -92,10 +95,7 @@ resource_usage:
   aws_docdb_cluster_instance.my_db:
     data_storage_gb: 1000         # Total storage for cluster in GB.
     monthly_io_request: 100000000 # Monthly number of input/output requests for cluster.
-    monthly_cpu_credit_hours: 100 # Monthly CPU credits used over the instance baseline in vCPU-hours, only applicable for T3 instances. 
-    
-  aws_docdb_cluster.my_cluster:
-    backup_storage_gb: 10000      # Amount of backup storage that is in excess of 100% of the storage size for the cluster in GB.
```

Comment:
did you mean to remove this and `aws_docdb_cluster`? I don't see the file being changed in the file

Reply:
Hi @alikhajeh1, I removed the duplicate entry for,  ```aws_docdb_cluster.my_cluster``` it was in the usage file twice.